src/runner.rs

```diff
@@ -203,32 +217,57 @@ impl<D: DB> Runner<D> {
         Ok(())
     }
 
+    /// Run a single record.
+    pub async fn run(&mut self, record: Record) -> Result<(), TestError> {
```

Comment:
This function is intended to be not async?
```suggestion
    pub fn run(&mut self, record: Record) -> Result<(), TestError> {
```

Reply:
Yes, my fault :(